### DISCUSSION TITLE
RNode bluetooth profile/connection to iOS?

### DISCUSSION THREAD

**Discussion by cobraPA** [markqvist/Reticulum#353] [General]
Hi All,

I moving toward getting RNode hardware to work on the iOS port of Sideband.  What profile is the bluetooth in the standard RNode firmware.  Does it support BLE or Standard?  I tried my Heltec and LilyGo nodes and none seem to be visible to the standard iOS Bluetooth pairing screen.  I did see their IDs sometimes with some basic scan software, so pairing may still be possible from Sideband.

**Comment by faragher** [comment]
I just want to verify you're going through all the steps required, because this bit me.

The LilyGo don't have a pairing button. Sure, it seems like they'll pair, but they won't. You'll need to connect over serial with rnodeconf and manually both enable Bluetooth and put it into pairing mode.  You can do this by calling:

```
rnodeconf [port] -b
rnodeconf [port] -p
```

This took me longer than I care to admit to figure out. Especially since I already paired one previously.

If you already did this and it doesn't work, I'll pull out the iPhone and see if I can help troubleshoot.

**Reply by cobraPA** [reply]
Yeah, I used a standalone computer to enable and set pairing mode on both the ones I tried.  It looks like maybe pairing mode enables bluetooth anyway?  But I tried both ways.  I did not see the RNodes appear in the iPhone bluetooth search list.  This is running newer iOS, 16.5.1 (c).

**Comment by cobraPA** [comment]
I think I've answered my question...  the current RNode is using Bluetooth Classic for connection to Android devices, however, iOS usually requires the use of BLE (or limits the number of recognized profiles for Bluetooth Classic devices.)  It looks like an extension to the RNode for BLE support is the way to go for use with iOS.
